editor/src/components/canvas/dom-walker.ts

```diff
@@ -729,14 +876,21 @@ function collectMetadata(
   cachedPaths: Array<ElementPath>
   collectedPaths: Array<ElementPath>
 } {
-  if (pathsForElement.length === 0) {
+  // Determine if the parent has been visited so far.
```

Comment:
please expand this to contain the 'why?'!

Reply:
Fixed in 2351319d3.